Makefile

```diff
@@ -122,10 +122,10 @@ build-simd-all: go.sum
 	docker cp -a latest-build:/home/builder/artifacts/ $(CURDIR)/
 
 build-simd-linux: go.sum $(BUILDDIR)/
-	$(if $(shell docker inspect -f '{{ .Id }}' cosmossdk/rbuilder 2>/dev/null),$(info found image cosmossdk/rbuilder),docker pull cosmossdk/rbuilder:latest)
+	docker pull cosmossdk/rbuilder:latest
```

Comment:
not needed, docker run will pull if the image is not present

Reply:
will remove in another pr